src/client/websocket/WebSocketShard.js

```diff
@@ -475,12 +475,17 @@ class WebSocketShard extends EventEmitter {
       return;
     }
     const hasGuildsIntent = new Intents(this.manager.client.options.intents).has(Intents.FLAGS.GUILDS);
-    // Step 2. Create a 15s timeout that will mark the shard as ready if there are still unavailable guilds
+    // Step 2. Create a timeout that will mark the shard as ready if there are still unavailable guilds
+    // * the timeout should be 15 seconds by default, but this can be optionally be changed in the client options via fetchGuildTimeout
+    // * a timeout time of zero will skip this timeout
+
+    const guildTimeoutAmount = this.manager.client.options.fetchGuildTimeout || 1500;
+
     this.readyTimeout = setTimeout(
       () => {
         this.debug(
           `Shard ${hasGuildsIntent ? 'did' : 'will'} not receive any more guild packets` +
-            `${hasGuildsIntent ? ' in 15 seconds' : ''}.\n   Unavailable guild count: ${this.expectedGuilds.size}`,
+            `${hasGuildsIntent ? `in ${guildTimeoutAmount} milliseconds` : ''}.\n   Unavailable guild count: ${this.expectedGuilds.size}`,
```

Comment:
```suggestion
            `${hasGuildsIntent ? ` in ${guildTimeoutAmount} milliseconds` : ''}.\n   Unavailable guild count: ${this.expectedGuilds.size}`,
```

Reply:
Should probably also say ms instead of milliseconds

Reply:
> Yeah I’m pretty sure that text you added goes over the 120 char limit, you should install ESLint to be aware of these things since the CI requires approval to run

thank you! I've corrected this issue now and it passed the lint :) @ImRodry 
I've learned a lot and I hope this PR is helpful to all discord js bot devs!